src/current/advisories/a131639.md

```diff
@@ -0,0 +1,211 @@
+---
+title: Technical Advisory 131639
+advisory: A-131639
+summary: During a sustained period of disk slowness in the presence of lease transfers, it is possible for some writes in a transaction that straddle multiple ranges to be lost.
+toc: true
+affected_versions: v23.1.0 to v23.1.26, v23.2.0 to v23.2.10, v24.1.0
+advisory_date: 2024-10-03
+docs_area: releases
+---
+
+Publication date: {{ page.advisory_date | date: "%B %e, %Y" }}
+
+## Description
+
+Since v22.2, through v24.2, in circumstances where there is a sustained period of disk slowness in the presence of lease transfers, it is possible for some writes in a transaction that straddle multiple ranges to be lost. In cases where these lost writes are not also present on one or more secondary indexes that could allow for reconstructing or repairing the row, these writes could be irrecoverably lost.
```

Comment:
```suggestion
In v22.2 through v24.2, in circumstances where there is a sustained period of disk slowness in the presence of lease transfers, it is possible for some writes in a transaction that straddle multiple ranges to be lost. In cases where these lost writes are not also present on one or more secondary indexes that could allow for reconstructing or repairing the row, these writes could be irrecoverably lost.
```
Slight edit here because I think the "Since...through..." construct reads a little confusing. I'm confused about the distinction being made between "lost" and "irrevocably lost", and I suspect in the next paragraph, the first and second sentences are saying the same thing.